src/menu/menu.c

```diff
@@ -67,6 +69,13 @@ static void menu_init (boot_params_t *boot_params) {
 
     sound_init_default();
 
+    mixer_ch_set_vol(SFX_CHANNEL, 0.5f, 0.5f);
+    wav64_open(&sfx_cursor, "rom:/cursorsound.wav64");
+    wav64_open(&sfx_exit, "rom:/back.wav64");
+    wav64_open(&sfx_settings, "rom:/settings.wav64");
+    wav64_open(&sfx_enter, "rom:/enter.wav64");
+    wav64_open(&sfx_error, "rom:/error.wav64");
+
```

Comment:
This can be moved into `sound_reconfigure`. Don't forget about a flag to do loading only once.